### PR TITLE
TST run and index elasticsearch in tests

### PR DESCRIPTION
**Pull request by jnothman**

### CODE REVIEW

**Comment by jnothman** [comment]
Current issues with testing in CI include:
* react-scripts not found in our reactivesearch container
* It seems elasticsearch is being forbidden to open a port.

Maybe it's not so easy to test a docker-compose-based webserver on GH workflows... Pity. Will leave this on the backburner. It was a nice idea to test the fuller pipeline... might need to seek advice from others.

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: 
+      - '**'
+
+jobs:
+  check-search-frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bring system up
+        run: |
```
Comment (hlydecker) [inline .github/workflows/frontend.yml:17]: Do we also need to make sure npm is installed prior to running docker-compose up?

Comment (jnothman) [reply .github/workflows/frontend.yml:17]: What doesn't make sense to me is that npm is run in the container, so what's available on the host machine is irrelevant.

**Comment by jnothman** [comment]
Changes to the docker setup here would be useful also in your frontend branch, @hlydecker

**Pull request merged**